app-misc/lcdproc/lcdproc-0.5.9.ebuild

```diff
@@ -0,0 +1,206 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="6"
+
+inherit autotools systemd
+
+DESCRIPTION="Displays real-time system information from your Linux/*BSD box on a LCD"
+HOMEPAGE="http://www.lcdproc.org/"
+SRC_URI="https://github.com/${PN}/${PN}/releases/download/v${PV}/${P}.tar.gz"
+
+KEYWORDS="~amd64 ~ppc ~x86"
+LICENSE="GPL-2"
```

Comment:
Verified it's not GPL-2+?

Reply:
Yes. All source files say:

```
 * This file is released under the GNU General Public License.
 * Refer to the COPYING file distributed with this package.
```

And COPYING says:

```
                    GNU GENERAL PUBLIC LICENSE
                       Version 2, June 1991

 Copyright (C) 1989, 1991 Free Software Foundation, Inc.
                          675 Mass Ave, Cambridge, MA 02139, USA
 Everyone is permitted to copy and distribute verbatim copies
 of this license document, but changing it is not allowed.
```